src/components/Dropdown/dropdown.css

```diff
@@ -2,8 +2,55 @@
 @import "src/css/variables/fonts.css";
 @import "src/css/variables/colors.css";
 
-.y-dropdown .ms-Dropdown-label{
+.y-dropdown .ms-Dropdown-label {
   @extend text__mediumSub;
   font-weight: $fontWeight__bold;
   color: $textColor__secondary;
 }
+
+.y-dropdown .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown-caretDownWrapper {
+  height: 36px;
+  line-height: 3.3rem;
+}
+
+.y-dropdown .ms-Dropdown-title {
+  color: $textColor;
+  border-color: $borderColor;
+}
+
+.y-dropdown .y-dropdown__caretDown {
+  color: $color__river;
```

Comment:
Can anyone advise what we should do here? I'd like to have some of these colors mapped to names in our colors file, but they didn't exist there yet. Thoughts on all of the colors I directly used from the palette?

Reply:
We seem to be using semantic color names everywhere else, so if there isn't a semantic color already, we should create one.

@mamuso @adrianmg is this a color we'll use just for carets (i.e. we name it `caretColor`) or will it be used for other types of things?

Reply:
you can just use $linkColor here, set it colors.css